apps/dokploy/templates/plausible/index.ts

```diff
@@ -23,7 +23,8 @@ export function generate(schema: Schema): Template {
 
 	const mounts: Template["mounts"] = [
 		{
-			mountPath: "./clickhouse/clickhouse-config.xml",
+			filePath: "clickhouse-config.xml",
+			mountPath: "../files/clickhouse/clickhouse-config.xml",
```

Comment:
```suggestion
			mountPath: "/clickhouse/clickhouse-config.xml",
```

Reply:
I found it will generate two empty directories after deploying (regardless of whether there is `../files`):
![image](https://github.com/user-attachments/assets/a91c7f44-23b8-4b15-b8f9-eb72c4c662d9)

Maybe it should be?:
```
filePath: "clickhouse/clickhouse-config.xml",
mountPath: "",
```

Reply:
May I ask what the "Mount path" here is used for?